### PR TITLE
Added End2End test for TF

### PR DESCRIPTION
**Pull request by pfinashx**
Finished E2E tests for TF part.

### CODE REVIEW

File: beta/requirements.txt [hunk 1]
```diff
@@ -7,6 +7,7 @@ tensorflow_addons==0.12.0
 tensorflow_hub
 texttable
 opencv-python
+pyyaml
```
Comment (alexsu52) [inline beta/requirements.txt:10]: It looks like a new dependency for the NNCF. @vshampor  Is it an approved dependency?

Comment (vshampor) [reply beta/requirements.txt:10]: It is not, and in fact it should not be used for security reasons. YAML parsing in general is insecure, so we should avoid having to handle .yml files ourselves.

Comment (pfinashx) [reply beta/requirements.txt:10]: This package is used to modify AC configs.
Now we have configs in beta/tests/tensorflow/data/ac_configs/ only for FP32 baselines models, and in tests we modify them on-the-fly for compressed models, this modification was discussed with developers by @daniil-lyakhov

Comment (vshampor) [reply beta/requirements.txt:10]: This requirement is not for the NNCF package then, but for the TF tests. Have the requirements.txt been separated for TF yet?

Comment (pfinashx) [reply beta/requirements.txt:10]: And in fact it used other OpenVino frameworks and tools, like training_extensions and Accuracy Checker.
I can separate req's and add requirements.txt in tests/ or we need add full configs for all models.

Comment (pfinashx) [reply beta/requirements.txt]: Updated

**Comment by pfinashx** [comment]
Jenkins please retry a build

**Comment by pfinashx** [comment]
Jenkins please retry a build

**Review by alexsu52 — APPROVED** [review]

**Comment by alexsu52** [comment]
@vshampor , please, merge if you have no comments.

**Pull request merged**